debs/Debian/amd64/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM debian:7
+FROM --platform=linux/amd64 debian:stretch
```

Comment:
For consistency, please use the flag `--platform` or use the format `architecture/debian:stretch` in all Debian Dockerfiles